publication/ver11/societal_impact.md

```diff
@@ -0,0 +1,216 @@
+Vocabulary used in the WoT Thing Description is based on English, much like
+HTML tags.  Support is included for including human-readable content in 
+multiple languages but it is up to device manufacturers to include that content.
+Access to the network interfaces of IoT devices would, of course, only be available
+to those with network access and a computer able to generate network requests.
+
+To broaden access, translation services are needed to be able to extend existing
+WoT Thing Models with third-party translations.  This aspect of the WoT Thing Models
+needs further work.  However, the WoT JP CG has been working on translating the
+specifications themselves into Japanese.
+
+### 6. [What effect may features of your specification have on minority groups?](https://w3ctag.github.io/societal-impact-questionnaire/#minority-groups)
+No specific impact, other than the language barrier noted.
+Minority groups may not be able to access metadata in their language.
+
+IoT devices or services may only be available to those able to afford access
+devices.  However, if IoT services are made available via a web front-end,
+a common pattern made easier to implement with the proposed WoT standards,
+then it would make IoT services as accessible as the general Web.
+
+### 7. [What are the power dynamics at play in implementations of your specification?](https://w3ctag.github.io/societal-impact-questionnaire/#power-dynamics)
+The WoT standard can be used to make details of how to access an IoT
+device available to a wider audience.  In particular, the manufacturer does
+not have to provide the metadata, it can be provided by a third party.  
+While this is not ideal ("official" support from a manufacturer would be more
+trustworth, third-party information would have to be used with care), it does
+mean that a manufacture will not be able to control access by restricting
+access to WoT Thing Descriptions.  They can simply be sourced from elsewhere,
+with some security caveats.
+
+The WoT also makes it easier to develop "local" IoT systems that do not
+require the cloud to function and can better manage private data.  They can
+in particular allow data from devices from different manufacturers to be used
+as part of a single system without that data having to go through a cloud
+API or service presented by that manufacturer.
+
+Finally, standardizing WoT metadata allows IoT devices from different manufacturers
+to be interchanged, reducing customer "lockin" to proprietary technologies or
+devices from a single manufacturer.
+
+In general, WoT gives users and the implementers of IoT systems more information
+about IoT devices and more control over how they are deployed.
+
+### 8. [What points of centralization does your feature bring to the web platform?](https://w3ctag.github.io/societal-impact-questionnaire/#centralization)
+This iteration of the WoT Architecture introduces WoT Discovery, which includes
```

Comment:
I would mention that the Discovery is an optional system that is no mandatory to run WoT-based applications

Reply:
done